docs/02-for-app-authors/10-verification.md

```diff
@@ -57,6 +57,7 @@ app ID prefixes will be matched:
   - **GitLab**: `io.gitlab.`
   - **GNOME GitLab**: `org.gnome.`
   - **KDE GitLab**: `org.kde.`
+  - **`pop-os` GitHub organization** `com.system76.`
```

Comment:
this is based on the login methods supported by the website so you can't list it here. in the team docs is fine.